state/validation_test.go

```diff
@@ -270,8 +273,7 @@ func TestValidateFailBlockOnCommittedEvidence(t *testing.T) {
 	// A block with a couple pieces of evidence passes.
 	block := makeBlock(state, height)
 	addr, _ := state.Validators.GetByIndex(0)
-	evpool.CommitEvidence(evpool.AddMockEvidence(height, addr))
```

Comment:
where did this go?

Reply:
I used the testify/mock on like 266 to return any IsCommitted function to true so I don't need to simulate this part of the evidence pool